### PR TITLE
Update thumbnail information

### PR DESCRIPTION
**Pull request by Qwerty-Space**

### CODE REVIEW

**Comment by Qwerty-Space** [comment]
I'm not actually sure what's going on with the resizing, so I'll remove that part

File: telethon/client/uploads.py [hunk 1]
```diff
@@ -201,9 +201,11 @@ async def send_file(
                 Optional JPEG thumbnail (for documents). **Telegram will
                 ignore this parameter** unless you pass a ``.jpg`` file!
 
-                The file must also be small in dimensions and in-disk size.
-                Successful thumbnails were files below 20kb and 200x200px.
-                Width/height and dimensions/size ratios may be important.
+                The file must also be small in dimensions and in disk size.
+                Successful thumbnails were files below 20kB and 320x320px.
+                For Telegram to accept a thumbnail, you must provide the
+                file's dimensions using either ``hachoir`` or
```
Comment (Lonami) [inline telethon/client/uploads.py:207]: Shouldn't this say "video dimensions"?

Comment (Lonami) [inline telethon/client/uploads.py:207]: I'm not a fan of "using `hachoir`". Users don't use this directly, it just needs to be installed.

File: telethon/client/uploads.py [hunk 2]
```diff
@@ -201,9 +201,11 @@ async def send_file(
...
+                :tl:`DocumentAttributesVideo`.
```
Comment (Lonami) [inline telethon/client/uploads.py:208]: Maybe better indicate that this should go in the list of `attributes`. Perhaps the examples section could be extended.

**Review by Lonami — CHANGES_REQUESTED** [review]

**Comment by Qwerty-Space** [comment]
Fixed in 9a6bc5ae72cbdeb15d9ac97b68bf7e8bab55ec0a

**Pull request closed without merging**